### PR TITLE
add `CONTRIBUTING.md`.

### PR DESCRIPTION
**Pull request by igalklebanov**
based on a comment by @koskimas on discord.

### CODE REVIEW

**Comment by koskimas** [comment]
Thanks once again!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Would love to contribute to Kysely?
+
+Awesome! We are not very organized yet. We don't have a plan or a roadmap.
+
+Probably the best way to get going is to watch the issue list and comment there if you see something you want to work on.
```
Comment (koskimas) [inline CONTRIBUTING.md:5]: Should add a mention here about the new `greenlit` tag.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+* Most features should have escape hatches.
+
+# Getting started
+
+1. fork kysely.
+
+2. clone your fork.
+
+3. install node.js (preferably LTS v18.x).
```
Comment (koskimas) [inline CONTRIBUTING.md:38]: preferable latest LTS

**Comment by igalklebanov** [comment]
Thanks for the feedback! 💯 

Made some changes around issues, functionality tests, etc.

**Pull request merged**